OfflineMapAreasExample/OfflineMapAreasExample/OfflineMapAreasExampleView.swift

```diff
@@ -16,13 +16,19 @@ import ArcGIS
 import ArcGISToolkit
 import SwiftUI
 
+@MainActor
 struct OfflineMapAreasExampleView: View {
     /// The map of the Naperville water network.
     @State private var map = Map(item: PortalItem.naperville())
     
     /// A Boolean value indicating whether the offline map ares view should be presented.
     @State private var isShowingOfflineMapAreasView = false
     
+    init() {
+        // Ask the job manager to schedule background status checks for every 30 seconds.
```

Comment:
It might be good to include a comment here that users can set this value to a value that they think reflects the nature of the jobs their app runs.